src/services/authService.js

```diff
@@ -1,71 +1,74 @@
 import axios from "../utils/axios";
 import { cfaSignIn, cfaSignOut } from "capacitor-firebase-auth";
-import {login,logout} from "../store/actions/accountActions"
-import {firebaseAuth} from "../firebase"
+import { login, logout } from "../store/actions/accountActions";
+import { firebaseAuth } from "../firebase";
+import "firebase/auth";
+import Auth from "../components/auth/Auth";
 
+export const setAxiosInterceptors = ({ onLogout }) => {
+  axios.interceptors.request.use(async (request) => {
+    if (firebaseAuth.currentUser) {
+      const accessToken = await firebaseAuth.currentUser.getIdToken();
+      request.headers.Authorization = accessToken;
+      return request;
+    }
+    return request;
+  });
+  axios.interceptors.response.use(
+    (response) => {
+      return response;
+    },
+    (error) => {
+      if (error.response && error.response.status === 401) {
+        removeSession();
 
- export const  setAxiosInterceptors = ({ onLogout }) => {
-    axios.interceptors.request.use(async(request)=>{
-      if(firebaseAuth.currentUser){
-        const accessToken = await firebaseAuth.currentUser.getIdToken();
-        request.headers.Authorization =  accessToken;
-        return request
-      }
-      return request
-    })
-    axios.interceptors.response.use((response) => {
-    return response
-  },
-      (error) => {
-        if (error.response && error.response.status === 401) {
-          removeSession();
-
-          if (onLogout) {
-            onLogout();
-          }
+        if (onLogout) {
+          onLogout();
         }
-        return Promise.reject(error);
       }
-    );
-  };
-
-
-
-  export const  loginAction = () => {
-    return  (dispatch)=>{
-      dispatch(login())
-      cfaSignIn("google.com").subscribe((user)=>{
-       setuserData(user.displayName,user.email,user.photoURL);
-       
-      });
+      return Promise.reject(error);
     }
-   
-  };
+  );
+};
 
-  export const  logoutAction = () => {
-    return (dispatch)=>{
-      dispatch(logout())
-      cfaSignOut().subscribe(()=>{
-       removeSession();
-      });
-    }
-   
-    
+export const loginAction = () => {
+  return (dispatch) => {
+    dispatch(login());
+    cfaSignIn("google.com").subscribe((user) => {
+      setuserData(user.displayName, user.email, user.photoURL);
+      user.sendEmailVerification();
```

Comment:
I think it's google signIn, this is mixing google signIn with email and password auth

Reply:
Hey i am really sorry , i do not know the user comes empty , can you please help

Reply:
i am really stuck

Reply:
Hey, Yash let's join on CFC discord under the project hangout section at 6 pm tomorrow I will explain the complete code base if it's fine.  

cc/ @Abhijay007  @yashvardhan05

Reply:
and we will discuss this issue also :)

Reply:
Yes , ok thank you